lib/Newsletter/Renderer/Columns/Renderer.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace MailPoet\Newsletter\Renderer\Columns;
 
+use MailPoet\Newsletter\Renderer\EscapeHelper as EHelper;
```

Comment:
I can't comment on unchanged lines but is all processed here in this file? What about `$content` in `renderOneColumn` and `renderMultipleColumns`? Is that handled by other blocks themselves? And what about `getMultipleColumnsContentStart`, nothing to handle there?

Reply:
$content should be processed by blocks themselves.